integrationTests/testInitializer.go

```diff
@@ -110,7 +109,7 @@ func createP2PConfig(initialPeerList []string) config.P2PConfig {
 			RandezVous:                       "/erd/kad/1.0.0",
 			InitialPeerList:                  initialPeerList,
 			BucketSize:                       100,
-			RoutingTableRefreshIntervalInSec: 2,
+			RoutingTableRefreshIntervalInSec: 100,
```

Comment:
100 even for the tests ?

Reply:
yes, this is the routing table refresh. The peers refresh list is done at about 2 seconds apart, this one just refreshes the peer information in the routing table. In tests, it could have been infinite, and it will worked as expected.